packages/bpk-component-image/src/BpkImage.js

```diff
@@ -77,25 +72,36 @@ class BpkImage extends React.Component {
           style={{ height: 0, paddingBottom: aspectRatioPc }}
           className={classNames.join(' ')}
         >
-          <div className={spinnerClassNames.join(' ')}>
-            <BpkSpinner />
-          </div>
+          <ReactCSSTransitionGroup
+            transitionName={{
+              leave: getClassName('bpk-image__shown'),
```

Comment:
`bpk-image` should be reserved for the outer most element i.e. the "block" in Block Element Modifier.

Here is a small guide:
```
.calculator {} // block 
.calculator--alternate {} // block modifier 
.calculator__definitions {} // element 
.calculator__definitions--expanded {} // element modifier
```